src/network/definitions/pay.ts

```diff
@@ -1 +1,2 @@
-export const PAY_HYDRATE_CHARGE = 'pay_hydrateCharge';
+export const CDP_HYDRATE_CHARGE = 'cdp_hydrateCharge';
+export const CDP_CREATE_PRODUCT_CHARGE = 'cdp_createProductCharge';
```

Comment:
Make a `constants.ts` file =?

Reply:
Ah I see we have one, should these be added to it?

Reply:
actually reverted this, it seems like a breaking change and we shouldn't

Reply:
lets stick with the `definitions/pay` file for now